# Пример обработки Ошибок!
num = input('Write your number ')
try:
    print(num + 10)
except Exception:
    print('Something goes wrong!')
